components/messenger.rst

```diff
@@ -142,21 +142,26 @@ through the transport layer, use the ``SerializerStamp`` stamp::
         ]))
     );
 
-At the moment, the Symfony Messenger has the following built-in envelope stamps:
+Here are some important envelope stamps that are shipped with the Symfony Messenger:
 
-#. :class:`Symfony\\Component\\Messenger\\Stamp\\SerializerStamp`,
-   to configure the serialization groups used by the transport.
-#. :class:`Symfony\\Component\\Messenger\\Stamp\\ValidationStamp`,
-   to configure the validation groups used when the validation middleware is enabled.
+#. :class:`Symfony\\Component\\Messenger\\Stamp\\DelayStamp`,
+   to delay handling of an asynchronous message.
+#. :class:`Symfony\\Component\\Messenger\\Stamp\\DispatchAfterCurrentBusStamp`,
+   to make the message be handled after the current buss has executed. Read more
```

Comment:
Double s on bus ;)

Reply:
I slipped in some Swedish. Sorry about that.

Reply:
And I don't know how it could be added, but if the treatment has been successful no?